lib/identity_cache/cached/recursive/association.rb

```diff
@@ -24,19 +24,40 @@ def #{cached_accessor_name}
           ParentModelExpiration.add_parent_expiry_hook(self)
         end
 
+        def read(record)
+          record.public_send(cached_accessor_name)
+        end
+
+        def write(record, records)
+          record.instance_variable_set(records_variable_name, records)
+        end
+
         def clear(record)
           if record.instance_variable_defined?(records_variable_name)
             record.remove_instance_variable(records_variable_name)
           end
         end
 
+        def fetch(records)
+          fetch_embedded(records)
+
+          records.flat_map(&cached_accessor_name.to_sym).tap(&:compact!)
```

Comment:
We should move the `.to_sym` to `Cached::Association#initialize` so that we aren't always converting it to a symbol at runtime.  It is only used as a string as part of the class eval strings.  Doesn't need to happen in this PR though.

Reply:
👌 , I'll make that change and update the `cached_association` call